src/Microsoft.CodeAnalysis.Analyzers/UnitTests/MetaAnalyzers/ReleaseTrackingAnalyzerTests.cs

```diff
@@ -545,8 +545,12 @@ class MyAnalyzer : DiagnosticAnalyzer
         [InlineData("", ReleaseTrackingHelper.TableHeaderNewOrRemovedRulesLine1 + BlankLine + "Id1 | Category1 | Warning |")]
         // Missing TableHeaderLine1 in unshipped
         [InlineData("", ReleaseTrackingHelper.TableTitleNewRules + BlankLine + ReleaseTrackingHelper.TableHeaderNewOrRemovedRulesLine2 + BlankLine + "Id1 | Category1 | Warning |", 2)]
+        // Missing TableHeaderLine1 in unshipped 2
```

Comment:
Instead of just having an opaque suffix `2`, can you add a more detailed comment that this is testing the regex matching with xyz aspects of the entry? Same for InlineData additions below.